### PR TITLE
HPCC-31384 Add release cadence to README.md

### PR DESCRIPTION
**Pull request by GordonSmith**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by GordonSmith** [comment]
@ghalliday you preview how it renders here:
image

File: README.md [hunk 1]
```diff
@@ -9,6 +9,41 @@ HPCC Systems offers an enterprise ready, open source supercomputing platform to
...
+    tickInterval 3month
+    dateFormat YYYY-MM-DD
+    section v8.12.x
+        Active:          done, 2022-04-01, 1y
+        Security:        done, 6M
+    section v9.0.x
+        Active:          done, 2022-10-01, 1y
+        Security:        active, 6M
+    section v9.2.x
+        Active:          active, 2023-04-01, 1y
```
Comment (ghalliday) [inline README.md:34]: 2023-07-01, 6M

File: README.md [hunk 2]
```diff
@@ -9,6 +9,41 @@ HPCC Systems offers an enterprise ready, open source supercomputing platform to
...
+gantt
+    title Release Schedule
+    axisFormat %Y-Q%q
+    tickInterval 3month
+    dateFormat YYYY-MM-DD
+    section v8.12.x
+        Active:          done, 2022-04-01, 1y
+        Security:        done, 6M
+    section v9.0.x
+        Active:          done, 2022-10-01, 1y
```
Comment (ghalliday) [inline README.md:31]: 2023-04-01, 9M

File: README.md [hunk 3]
```diff
@@ -9,6 +9,41 @@ HPCC Systems offers an enterprise ready, open source supercomputing platform to
...
+---
+displayMode: compact
+---
+gantt
+    title Release Schedule
+    axisFormat %Y-Q%q
+    tickInterval 3month
+    dateFormat YYYY-MM-DD
+    section v8.12.x
+        Active:          done, 2022-04-01, 1y
```
Comment (ghalliday) [inline README.md:28]: 2023-02-07, 6M

File: README.md [hunk 4]
```diff
@@ -9,6 +9,41 @@ HPCC Systems offers an enterprise ready, open source supercomputing platform to
...
+        Security:        done, 6M
```
Comment (ghalliday) [inline README.md:29]: 9M

File: README.md [hunk 5]
```diff
@@ -9,6 +9,41 @@ HPCC Systems offers an enterprise ready, open source supercomputing platform to
...
+    section v9.0.x
+        Active:          done, 2022-10-01, 1y
+        Security:        active, 6M
+    section v9.2.x
+        Active:          active, 2023-04-01, 1y
+        Security:        6M
+    section v9.4.x
+        Active:          active, 2023-10-01, 1y
```
Comment (ghalliday) [inline README.md:37]: , 9M

**Review by ghalliday — COMMENTED** [review]
Looks good.  Some suggested changes to the dates.

File: README.md [hunk 6]
```diff
@@ -9,6 +9,41 @@ HPCC Systems offers an enterprise ready, open source supercomputing platform to
...
+        Active:          done, 2023-04-01, 9M
+        Security:        active, 6M
+    section v9.2.x
+        Active:          active, 2023-07-01, 6M
+        Security:        6M
+    section v9.4.x
+        Active:          active, 2023-10-01, 9M
+        Security:        6M
+    section v9.6.x
+        Active:          2024-04-01, 1y
```
Comment (ghalliday) [inline README.md:40]: , 6M and also line 43.  Standard is really 6M active, 6M security, but it has changed a bit around the 9.4.x release.

File: README.md [hunk 7]
```diff
@@ -9,6 +9,41 @@ HPCC Systems offers an enterprise ready, open source supercomputing platform to
 * [Installation and Running](https://hpccsystems.com/training/documentation/installation-and-administration)
 * [Developer Documentation](https://hpcc-systems.github.io/HPCC-Platform/)
 
+# Release + Support Policy
+
+In general, a new version of the HPCC Platform is released every 6 months. These releases can be either Major (with breaking changes) or Minor (with new features). Maintenance and security releases (point releases) are typically made weekly, and may occasionally include technical previews.
```
Comment (ghalliday) [inline README.md:14]: Every 3 months

**Review by ghalliday — COMMENTED** [review]
@GordonSmith a couple of final comments

**Review by ghalliday — CHANGES_REQUESTED** [review]
Active is still too long for 9.6 and 10.0.0.  Also 10.0.0 is more likely to be 9.8.

**Comment by GordonSmith** [comment]
@ghalliday, @cloLN - updated to include 9.8.x and 9.10.x

**Comment by cloLN** [comment]
Looks good.  thanks.

**Review by cloLN — APPROVED** [review]
Changes look good.

**Review by ghalliday — APPROVED** [review]

**Pull request merged**